### PR TITLE
Create health_data.md

### PR DESCRIPTION
**Pull request by smza**
Transferring the content for Health data page from  a google doc

### CODE REVIEW

**Comment by bedroesb** [comment]
Maybe related to: https://github.com/elixir-europe/rdmkit/issues/753

**Comment by lauportell** [comment]
I just reviewed the document and it looks great. My only comment is that it would be good to mention the specific aspects of the use of EHR data for secondary use, which involves using this data for purposes beyond the initial clinical care, such as research or analytics. Like how this data should be adapted for it or if there's need of processing (anonymisation?) It would be good to add information about the HealthData@EU project, for example? This addition would contribute to a more general understanding of the management and ethical use of health data in research studies. Maybe it's out of the scope of this page, let me know what you think.

**Comment by smza** [comment]
> I just reviewed the document and it looks great. My only comment is that it would be good to mention the specific aspects of the use of EHR data for secondary use, which involves using this data for purposes beyond the initial clinical care, like research. For example, how this data should be adapted for it or if there's need of processing (anonymisation?). It would be good to add information about the HealthData@EU project, for example? This would contribute to a general understanding of the management and ethical use of health data in research studies. What do you think?
> 
> If you agree, I can give it a try to add this information.

Hi @lauportell , thanks for your valuable comment. To add this information I think to is best to discuss this with the authors of the page mainly Tim Beck (from Health Data Focus group). I'll write an email and copy you in since I don't have Tim's gitid.

**Comment by lauportell** [comment]
Hi all!
I have added the information about TEHDAS and HealthData@EU in the page. Let me know if you think it fits there. I agree with Tim's comment about leaving the anonymisation piece for the sensitive data page as it is already mentioned there.
Thank you all for the good work!

**Review by lauportell — APPROVED** [review]

**Comment by lauportell** [comment]
I just approved again the PR ( didn't know I could do it as I did some changes :) )
Small things to be done before merging:
1) Add contributors to the contributors yaml
2) Add tools to the tools table and change to the new way of visualizing tools
3) Add page to the sidebar
4) Add news item
@smza will you do it?

**Comment by smza** [comment]
> I just approved again the PR ( didn't know I could do it as I did some changes :) ) Small things to be done before merging:
> 
> 1. Add contributors to the contributors yaml
> 2. Add tools to the tools table and change to the new way of visualizing tools
> 3. Add page to the sidebar
> 4. Add news item
>    @smza will you do it?

Hi @lauportell thanks for your contributions.  I'm waiting from the group for information/description for the tools before adding them to yaml. Once I get that then I'll add the tools to the yaml file.

Cheers
Munazah

File: pages/your_domain/health_data.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+
+## Patient-generated health data from questionnaires 
+ 
+### Description
+Participants’ health data which can be collected via surveys, interviews, and monitoring, are called Clinical Outcome Measures (COMs). In surveys, the questionnaire is either completed by the participants or their representative (e.g., family member or caregiver). In interviews, a healthcare professional asks/explains the questionnaire to the participants and records their responses. Monitoring includes measuring a participant’s behaviour or activity in both a clinical and non-clinical setup. Based on this, COMs can be classified into four broad categories:
+Patient Reported Outcome Measure (PROM) is a measurement that comes directly from the patient (i.e., study participant) about the status of a patient’s health condition without amendment or interpretation of the patient’s response by a clinician or anyone else.
```
Comment (bianchini88) [inline pages/your_domain/health_data.md:23]:
```suggestion
Patient Reported Outcome Measure (PROM) is a measurement that comes directly from the patient (i.e. study participant) about the status of a patient’s health condition without amendment or interpretation of the patient’s response by a clinician or anyone else.
```

File: pages/your_domain/health_data.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+Observer-Reported Outcome Measure (ObsROM) is a measurement based on a report of observable signs, events or behaviours related to a patient’s health condition by someone (e.g., family member or caregiver) other than the patient or a health professional.
```
Comment (bianchini88) [inline pages/your_domain/health_data.md:24]:
```suggestion
Observer-Reported Outcome Measure (ObsROM) is a measurement based on a report of observable signs, events or behaviours related to a patient’s health condition by someone (e.g. family member or caregiver) other than the patient or a health professional.
```

File: pages/your_domain/health_data.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+To collect health data directly from participants, an efficient data collection strategy will consider the following components:
+* Consider the impact of the digital divide when digital collection methods are utilised. 
+* Define key scientific objectives with patients and stakeholders from clinical, social-sciences and data analysis domains.
+* In the case of a longitudinal study, define time points for the data collection (consult doctors on typical visit schedules to minimise the burden on participants and a calendar for holidays and other events).
+* Any personally identifiable information (PII), including communication details like name, address, email, or telephone number, needs to be separated from data and stored in a separate system with additional security measures like Two-Factor Authentication and encryption at rest. 
+* Design the privacy policy for the survey according to relevant local guidelines with the information governance/data protection officer and IT team
+* Design the questionnaires with consideration given to interoperability (e.g., mapping questions/answers to concepts in clinical terminologies) and avoiding ambiguity in the questions.
```
Comment (bianchini88) [inline pages/your_domain/health_data.md:38]:
```suggestion
* Design the questionnaires with consideration given to interoperability (e.g. mapping questions/answers to concepts in clinical terminologies) and avoiding ambiguity in the questions.
```

File: pages/your_domain/health_data.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+  * Guidelines: 
+     - [EC Guidelines for the development and criteria for the adoption of Health Survey instruments] (https://ec.europa.eu/health/ph_information/dissemination/reporting/healthsurveys_en.pdf)
+     - [EC Selection of a Coherent Set of Health Indicators for the European Union Phase II](https://ec.europa.eu/health/ph_projects/2000/monitoring/fp_monitoring_2000_frep_03_en.pdf)
+  * Check the readability and accessibility of the questionnaires.
+  * Use validated, standardised questionnaires where possible, for example REDCap has a [library of data collection instruments](https://redcap.vanderbilt.edu/consortium/library/search.php)
+  * Use a collaborative editing platform with versioning 
+  * Finalise the questionnaires in the required template (if any) for ethical approval
+
+* Development of the machine readable data dictionary for all languages
```
Comment (bianchini88) [inline pages/your_domain/health_data.md:48]:
```suggestion
* Development of the machine-readable data dictionary for all languages:   
```

**Review by bianchini88 — CHANGES_REQUESTED** [review]

**Comment by bedroesb** [comment]
@smza do you have a look a the review from @bianchini88 ?

**Comment by smza** [comment]
> @smza do you have a look a the review from @bianchini88 ?
not yet

@bedroesb I'm waiting for review of some new content that was added by Laura from the team. The team also have to provide the Tools descriptions. probably end of feb

File: pages/your_domain/health_data.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
+---
+title: Human health data
+search_exclude: true
```
[the remark this answers is not in the document]
Comment (bedroesb) [reply pages/your_domain/health_data.md:3]: yes indeed!

**Review by smza — COMMENTED** [review]
Accepted all changes

File: _data/CONTRIBUTORS.yaml [hunk 6]
```diff
@@ -610,3 +610,36 @@ Adriano Fonzino:
...
+Salvador Capella-Gutierrez:
+    email: salvador.capella@bsc.es 
+    affiliation: Barcelona Supercomputing Centre (BSC) 
+Magda Chegkazi:
+    email: magda.chegkazi@elixir-europe.org 
+    affiliation: ELIXIR 
+Arshiya Merchant:
+    email: arshiya.merchant@elixir-europe.org
+    affiliation: ELIXIR 
+Carlos Luís Parra Calderón:
```
Comment (bedroesb) [inline _data/CONTRIBUTORS.yaml:642]: @smza still to be filled in or you can take them out :)

**Review by bianchini88 — APPROVED** [review]

File: _data/CONTRIBUTORS.yaml [hunk 7]
```diff
@@ -610,3 +610,36 @@ Adriano Fonzino:
     git: F0nz0
     orcid: 0009-0009-3325-8554
     affiliation: University of Bari
+Tim Beck:
+    git: tb143
+    email: tim.beck@nottingham.ac.uk
+    orcid: 0000-0002-0292-7972
+    affiliation: University of Nottingham
+Marcus Buchwald:
```
Comment (bianchini88) [inline _data/CONTRIBUTORS.yaml:618]: Most new contributors do not have ORCID.

**Comment by bedroesb** [comment]
Question to all: did we tackle https://github.com/elixir-europe/rdmkit/issues/753 ?

File: pages/your_domain/health_data.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+
+### Solutions
+GDPR compliant EDC systems that support the capture of COMs:
+* {% tool "redcap" %} - Academic licence
+    - Check the licence terms carefully, as some uses in commercial research may not be allowed.
+    - REDCap – [online training videos](https://tess.elixir-europe.org/materials?tools=REDCap)
+* {% tool "redcap-cloud" %} - Commercial licence
+    - A fully managed REDCap system based in the cloud, provided by nPhase.
+* {% tool "castor" %} - Academic/commercial licence
+* {% tool "eu-survey" %} - SaaS provided by EU commission
```
Comment (bianchini88) [inline pages/your_domain/health_data.md:69]:
```suggestion
* {% tool "eu-survey" %} - SaaS provided by the EU Commission
```

File: pages/your_domain/health_data.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+
+For member states within the EU, the European Commission published a proposed regulation for establishing the [European Health Data Space (or “EHDS”)](https://health.ec.europa.eu/ehealth-digital-health-and-care/european-health-data-space_en). The proposed regulation establishes a common space for health data where:
+  * Individuals are empowered through increased digital access to and control of their electronic personal health data
+  * Researchers, innovators and policymakers can request access to these electronic health data in a trusted and secure way that preserves the individual’s personal data
+The EHDS will also regulate the interoperability of EHRs to a certain extent, in particular to facilitate the exchange of the patient summary data.
+
+EHDS places a significant emphasis on promoting the secondary use of health data, to serve research, innovation, policy making and regulatory purposes. To achieve this, several European initiatives have been launched to establish its foundation and define guiding principles. Among these endeavors, two signification projects stand out:
```
Comment (bianchini88) [inline pages/your_domain/health_data.md:97]:
```suggestion
EHDS places a significant emphasis on promoting the secondary use of health data, to serve research, innovation, policy-making and regulatory purposes. To achieve this, several European initiatives have been launched to establish its foundation and define guiding principles. Among these endeavors, two signification projects stand out:
```

File: pages/your_domain/health_data.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+  * Regulations: [Health Insurance Portability and Accountability Act (HIPAA) and Health Information Technology for Economic and Clinical Health (HITECH)](https://www.hhs.gov/hipaa/for-professionals/security/laws-regulations/index.html)
+
+The [International Patient Summary (IPS)](https://international-patient-summary.net/) is a standardised set of basic patient-related physiological and clinical data that includes the most important health and care related facts required to ensure safe and secure healthcare. It comprises data about medications, allergies/intolerances, problems, immunizations, results and procedures for a specified patient. It is a joint standard of five standard development organisations (CEN, HL7, IHE, ISO and SNOMED) and is actively supported by the Global Digital Health Partnership and the World Health Organisation.
+
+### Solutions
+
+It is likely that the use of one of the following standards will not be a choice, but something imposed by the software providers of the EHR systems.These interchange formats are the standards that are likely to be present as part of the EHR, and include:
```
Comment (bianchini88) [inline pages/your_domain/health_data.md:108]:
```suggestion
Likely, the use of one of the following standards will not be a choice, but something imposed by the software providers of the EHR systems. These interchange formats are the standards that are likely to be present as part of the EHR, and include:
```

File: _data/CONTRIBUTORS.yaml (hunk 6 above)
Comment (smza) [reply _data/CONTRIBUTORS.yaml:642]: This is all the information I have for the contributors for now

File: pages/your_domain/health_data.md (hunk 10 above)
Comment (smza) [reply pages/your_domain/health_data.md:108]: I would prefer to keep the original text.

Comment (bianchini88) [reply pages/your_domain/health_data.md:108]: I have edited the suggestion above.

File: _data/CONTRIBUTORS.yaml (hunk 7 above)
Comment (smza) [reply _data/CONTRIBUTORS.yaml:618]: Yes thats the way it's going to be for now.  They don't have time to chase the authors and the page is needed live asap!

**Review by smza — COMMENTED** [review]
Review done! All suggestions have been resolved.

**Pull request merged**